src/agent/debuglet.ts

```diff
@@ -266,6 +266,21 @@ export class Debuglet extends EventEmitter {
     return extend(true, {}, defaultConfig, config, envConfig);
   }
 
+  static async findFiles(shouldHash: boolean, baseDir: string): Promise<{
```

Comment:
nit: not very readable; a named type would be better. Is it needed though? Doesn't the compiler infer these from the return type?

Reply:
I've added a named type.  The compiler can infer the return type, but I need to reference the type when I create a `findResults` variable.